### PR TITLE
Update team section

### PR DESCRIPTION
**Pull request by KarthikSiddarth**
# Resolves

- [AR-1413](https://team-1624093970686.atlassian.net/browse/AR-1413)

## Changes

1. Added new team members.
2. Updated Rohit's designation. [AR-1430](https://team-1624093970686.atlassian.net/browse/AR-1430)
3. Updated Lakshmikanth's social links.

# Screenshots
Screenshot 2022-02-08 at 4 37 10 PM

# Checklist

- [X] The branch name follows the format: `developer-name/AR-XXX-issue-name`.
- [X] The changes have been tested locally.

### CODE REVIEW

File: components/about/OurTeam.vue [hunk 1]
```diff
@@ -149,8 +149,8 @@ export default {
           designation: 'DevOps',
           image: 'Lakshmikanth',
           links: {
-            linkedin: 'linkedin.com/in/lakshmikanth-bharathi-2a4754b8',
-            twitter: 'https://twitter.com/LakshmikanthBh8',
+            linkedin: 'http://linkedin.com/in/lakshmikanth-bharathi-2a4754b8',
```
Comment (ajithranka) [inline components/about/OurTeam.vue:152]: [nit] Use `https` for consistency.

File: components/about/TeamMember.vue [hunk 2]
```diff
@@ -123,6 +123,7 @@ export default {
 
 .member-image > * {
   width: 16rem;
+  border-radius: 50%;
```
Comment (ajithranka) [inline components/about/TeamMember.vue:126]: The images have been grouped now on Figma and you should be able to export them with the circular masking. 

This works too, so no issues if you'd like to keep the current changes.

**Review by ajithranka — APPROVED** [review]

File: components/about/OurTeam.vue (hunk 1 above)
Comment (KarthikSiddarth) [reply components/about/OurTeam.vue]: done, [commit link](https://github.com/arcana-network/website/pull/84/commits/2b91f87f700cb4e6a66adae544b0785d77b361e0)

**Review by shrinathprabhu — APPROVED** [review]
LGTM

**Pull request merged**